stanford-sw/translation_maps/library_on_order_map_orig.properties

```diff
@@ -0,0 +1,34 @@
+# Map ON-ORDER Library Codes to User Friendly Description - Long Version for Facet Values
```

Comment:
The original file came from Jvine so I thought we needed to keep it in case there was a question later.